osu.Game/Overlays/Dialog/PopupDialogDangerousButton.cs

```diff
@@ -92,34 +88,28 @@ protected override void OnMouseUp(MouseUpEvent e)
             {
                 if (!e.HasAnyButtonPressed)
                 {
-                    stopTickSound();
                     AbortConfirm();
                 }
             }
 
-            private void playTick(double progress)
+            private void progressChanged(ValueChangedEvent<double> progress)
             {
-                lowPassFilter.CutoffTo((int)(Progress.Value * AudioFilter.MAX_LOWPASS_CUTOFF * 0.5f));
-
-                tickSample.Frequency.Value = 1 + progress * 0.5f;
-                tickSample.Volume.Value = 0.5f + progress / 2f;
-                tickSample.Play();
+                if (progress.NewValue < progress.OldValue)
+                {
+                    lowPassFilter.CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF);
```

Comment:
I'm not sure this is necessary? It also causes clicking when restarting the confirmation repeatedly.

https://user-images.githubusercontent.com/272140/173778107-e12abd41-0c74-4e8b-a6ee-5a4f9fef45f5.mov

Reply:
The alternative will be to have it track the progress at all times, ie.

```csharp
            private void progressChanged(ValueChangedEvent<double> progress)
            {
                lowPassFilter.CutoffTo((int)(progress.NewValue * AudioFilter.MAX_LOWPASS_CUTOFF * 0.5));

                if (progress.NewValue < progress.OldValue)
                    return;

                if (Clock.CurrentTime - lastTickPlaybackTime >= 30)
                {
                    tickSample.Frequency.Value = 1 + progress.NewValue * 0.5f;
                    tickSample.Volume.Value = 0.5f + progress.NewValue / 2f;
                    tickSample.Play();

                    lastTickPlaybackTime = Clock.CurrentTime;
                }
            }
```

Reply:
Could also just reset the filter in `OnMouseUp()` like in `Confirm()`?

Reply:
Sure, I'll do one of those.